### PR TITLE
Fixes AIize

### LINKED ISSUE #67028: 'Make AI' button in VV dropdown does not work
Reporting client version: 514.1583

## Round ID:
[183219](https://scrubby.melonmesa.com/round/183219)

## Testmerges:
- [refactors statpanel to use tgui API](https://www.github.com/tgstation/tgstation/pull/66971)
- [Adds Cargorilla](https://www.github.com/tgstation/tgstation/pull/67003)
- [Fuck it, we ball. Adds a very simple balling skill and minorly reworks dunking and shooting, adds ball stealing](https://www.github.com/tgstation/tgstation/pull/66960)

## Reproduction:

xtnrhTOJwz

player in question was in thunderdome, spawned in as a human, and had already been an AI earlier in the shift. I was also human in the tdome.

### PR DESCRIPTION
**Pull request by Time-Green**
closes #67028

:cl:
fix: Fixes AIize not working
/:cl:

Params were inverted for anything carbons and humans

### CODE REVIEW

**Comment by Sealed101** [comment]
does it close these two #68643 #67028 ?

**Comment by Time-Green** [comment]
This fix only affects carbons, so this fixes only the second one. I can check in a bit to see what’s wrong with ghosts

File: code/modules/mob/transform_procs.dm [hunk 1]
```diff
@@ -114,7 +114,7 @@
 	invisibility = INVISIBILITY_MAXIMUM
 	return ..()
 
-/mob/living/carbon/human/AIize(transfer_after = TRUE, client/preference_source)
+/mob/living/carbon/human/AIize(client/preference_source, transfer_after = TRUE, )
```
Comment (san7890) [inline code/modules/mob/transform_procs.dm:117]:
```suggestion
/mob/living/carbon/human/AIize(client/preference_source, transfer_after = TRUE)
```

**Review by san7890 — CHANGES_REQUESTED** [review]

File: code/modules/mob/transform_procs.dm (hunk 1 above)
Comment (Time-Green) [reply code/modules/mob/transform_procs.dm]: I really did do that huh

**Pull request merged**